### ISSUE TITLE
"undefined symbol: _ZN3c105ErrorC1ENS_14SourceLocationERKSs

### ISSUE THREAD

**Issue by wang99711123** [baowenbo/DAIN#47]
Hi.Thanks for the wonderful work!

I copied the virtual environment using “environment.yaml”which is provided.
And I execute build.sh successfully.

Now facing  below the issue:
undefined symbol: _ZN3c105ErrorC1ENS_14SourceLocationERKSs

Any  suggestion？

**Comment by laomao0** [comment]
This may be your CUDA version is not compilable with your PyTorch version.
Check your cuda version by:
$ python
$ import torch
$ print('torch.version.cuda')

then check your installed pytorch version supports the cuda or not?
If not, re-install the pytorch, then re-compile the cuda codes in my_packages.

**Comment by wang99711123** [comment]
Got it！Thank you for help！
And I am trying to reproduce your experimental results with the guide of Readme
But the IE and PSNR of the model I trained are not as good as the pre-trained model.
Do I need to resize the batch？

**Comment by wang99711123** [comment]
> 
> 
> This may be your CUDA version is not compilable with your PyTorch version.
> Check your cuda version by:
> $ python
> $ import torch
> $ print('torch.version.cuda')
> 
> then check your installed pytorch version supports the cuda or not?
> If not, re-install the pytorch, then re-compile the cuda codes in my_packages.

Got it！Thank you for help！
And I am trying to reproduce your experimental results with the guide of Readme.
CUDA_VISIBLE_DEVICES=0 python train.py --datasetPath /path/to/your/dataset --batch_size 1 --save_which 1 --lr 0.0005 --rectify_lr 0.0005 --flow_lr_coe 0.01 --occ_lr_coe 0.0 --filter_lr_coe 1.0 --ctx_lr_coe 1.0 --alpha 0.0 1.0 --patience 4 --factor 0.2
But the IE and PSNR of the model I trained are not as good as the pre-trained model.
Do I need to resize the batch？
Thank you for your reading!

**Comment by laomao0** [comment]
maybe you should pre-load the pre-trained model, then setting a small learning rate to train the model on your own datasets.

**Comment by laomao0** [comment]
> 



> > This may be your CUDA version is not compilable with your PyTorch version.
> > Check your cuda version by:
> > $ python
> > $ import torch
> > $ print('torch.version.cuda')
> > then check your installed pytorch version supports the cuda or not?
> > If not, re-install the pytorch, then re-compile the cuda codes in my_packages.
> 
> Got it！Thank you for help！
> And I am trying to reproduce your experimental results with the guide of Readme.
> CUDA_VISIBLE_DEVICES=0 python train.py --datasetPath /path/to/your/dataset --batch_size 1 --save_which 1 --lr 0.0005 --rectify_lr 0.0005 --flow_lr_coe 0.01 --occ_lr_coe 0.0 --filter_lr_coe 1.0 --ctx_lr_coe 1.0 --alpha 0.0 1.0 --patience 4 --factor 0.2
> But the IE and PSNR of the model I trained are not as good as the pre-trained model.
> Do I need to resize the batch？
> Thank you for your reading!

maybe you should pre-load the pre-trained model, then setting a small learning rate to train the model on your own datasets.
